pandas/core/frame.py

```diff
@@ -3843,10 +3843,15 @@ def _series(self):
     def lookup(self, row_labels, col_labels) -> np.ndarray:
         """
         Label-based "fancy indexing" function for DataFrame.
-
         Given equal-length arrays of row and column labels, return an
         array of the values corresponding to each (row, col) pair.
 
+        .. deprecated:: 1.2.0
+            DataFrame.lookup is deprecated,
```

Comment:
i think you need a blank line after the deprecated

Reply:
Not sure if that's needed looking at the other deprecated messages in `Series.py`. I think I solved it by removing the line break in the summary above the deprecated message.